staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/object/matcher.go

```diff
@@ -36,7 +36,7 @@ func matchObject(obj runtime.Object, selector labels.Selector) bool {
 	}
 	accessor, err := meta.Accessor(obj)
 	if err != nil {
-		klog.V(5).Infof("cannot access metadata of %v: %v", obj, err)
+		klog.V(5).InfoS("access metadata failed", "obj", obj, "error", err)
```

Comment:
```suggestion
		klog.V(5).InfoS("Access metadata failed", "object", klog.KObj(obj), "err", err)
```

Reply:
From https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message
```
* Use active voice. Use complete sentences when there is an acting subject ("A could not do B") or omit the subject if the subject would be the program itself ("Could not do B").
```
Who is actor in this situation? Could we change sentence to use active voice?

Reply:
The "obj" here is type of , not kubernetes objects. So could not use klog.KObj().